### DISCUSSION TITLE
Link does not get prefetched

### DISCUSSION THREAD

**Discussion by p-christ** [vercel/next.js#26369] [Help] [answered]
My understanding is that anything wrapped by Link should be preloaded by Next JS but by looking at the chrome dev tools Network tab I can see that this is not happening. Does anyone know why that might be?

I get this behaviour by just creating a next app as in the tutorial

```
npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"
```

then creating a new route page called "hello" with just a div in it and some text, then i add this code to the home page

```
<Link href="/hello">
  <a>PRESS ME</a>
</Link>
```

Then chrome dev tools tells me that when loading the home page that the "hello" page does not also get preloaded.

**Comment by jamesmosier** [comment, marked as answer]
Are you running the app in production mode or via `npm run dev`? In development, pages are not prefetched.

**Reply by p-christ** [reply]
thank you, i was running it in dev mode but can see now that it works in production mode, thanks!
